__tests__/index.test.js

```diff
@@ -477,9 +477,72 @@ describe("GET /api/users", () => {
         const { body } = response;
         const { users } = body;
         const length = users.length;
-        const {userData} = data
+        const { userData } = data;
         expect(length).toBe(4);
-        expect(users).toMatchObject(userData)
+        expect(users).toMatchObject(userData);
+      });
+  });
+});
+describe("GET /api/articles?topic=", () => {
+  it("GET 200: should accept a topic query, and respond with a filtered set of articles according to it", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        const { articles } = body;
+        const length = articles.length;
+        expect(length).toBe(4);
+
+        articles.forEach((article) => {
+          expect(Object.keys(article).length).toBe(8);
```

Comment:
Should you wish to add new keys to articles in the future this test would break. I'd avoid such brittle assertions that your objects are _always_ going to have 8 keys.